-1

Tested at: 49ec2c0dc51564966d4d0c6217ef4a91c28bb94f

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17534/17815/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-02-15-2300/src/RecoLocalCalo/HcalRecAlgos/src/PedestalSub.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-02-15-2300/src/RecoLocalCalo/HcalRecAlgos/src/HcalSeverityLevelComputerRcd.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-02-15-2300/src/RecoLocalCalo/HcalRecAlgos/src/AbsPlan1RechitCombiner.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-02-15-2300/src/RecoLocalCalo/HcalRecAlgos/src/HcalSeverityLevelComputer.cc 
In file included from /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-02-15-2300/src/RecoLocalCalo/HcalRecAlgos/src/SimpleHBHEPhase1Algo.cc:7:0:
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-02-15-2300/poison/RecoLocalCalo/HcalRecAlgos/interface/HBHERecHitAuxSetter.h:1:2: error: #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
 #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
  ^
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-02-15-2300/src/RecoLocalCalo/HcalRecAlgos/src/HcalHF_S9S1algorithm.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-02-15-2300/src/RecoLocalCalo/HcalRecAlgos/src/HBHEStatusBitSetter.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-02-15-2300/src/RecoLocalCalo/HcalRecAlgos/src/HBHEPulseShapeFlag.cc 
</pre>

--- a/RecoLocalCalo/HcalRecAlgos/src/SimpleHBHEPhase1Algo.cc
+++ b/RecoLocalCalo/HcalRecAlgos/src/SimpleHBHEPhase1Algo.cc
@@ -4,10 +4,10 @@
 
 #include "RecoLocalCalo/HcalRecAlgos/interface/SimpleHBHEPhase1Algo.h"
 #include "RecoLocalCalo/HcalRecAlgos/interface/HcalCorrectionFunctions.h"
-#include "RecoLocalCalo/HcalRecAlgos/interface/HBHERecHitAuxSetter.h"
 
 #include "FWCore/Framework/interface/Run.h"
 
+#include "DataFormats/HcalRecHit/interface/HBHERecHitAuxSetter.h"
 #include "DataFormats/METReco/interface/HcalPhase1FlagLabels.h"
 
 

